scripts/install

```diff
@@ -52,6 +52,7 @@ export WITH_EXTRA_TOOLS
 export WITH_WATCHDOG
 export WITH_YRD
 export WITH_SSB
+export WITH_SSB_PATCHFOO
 export WITH_SSB_WEB
```

Comment:
No we don't. 
I changed it to SSB_WEB_PI.

SSB_WEB_PI is a php client that executes sbot in the background (really slow) to access APIs. It has the advantage of storing login/password credentials to generated sbot identity so multiple identities can exist in on SBOT instance depending what L/P you use.

Because it has multiple identities it needed an additional local discovery service to broadcast those new identities. Otherwise you will only see the "pub" identify not the newly generated ones.